src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1359,6 +1443,30 @@ components:
         - registeredAddress
         - delivery
 
+    ReIssueAnonymousElectorDocumentRequest:
+      title: ReIssueAnonymousElectorDocumentRequest
+      description: Request body to re-issue an Anonymous Elector Document (AED)
+      type: object
+      properties:
+        sourceType:
+          $ref: '#/components/schemas/SourceType'
```

Comment:
I'm not sure about `sourceType` (it's an enum of `voter-card` or `anonymous-elector-document`) ... 🤔 

Half of me says we don't need it as it's implied through the API path .... but there is a semi-precedent set where we already use it in `GenerateAnonymousElectorDocumentRequest` and `GenerateTemporaryCertificateRequest` 🤔 

What do we think? I guess the possible difference here is that the REST APIs that use `GenerateAnonymousElectorDocumentRequest` and `GenerateTemporaryCertificateRequest` are meant to be called by VCA, rather than by the UI, so we can be confident that VCA sets the correct value
Whereas this API is meant to be called by the UI .....

Thoughts anyone ?

Reply:
I agree. I don't think we need `sourceType` at all for generating AEDs. It's implicitly `ANONYMOUS_ELECTOR_DOCUMENT`

Reply:
Done